src/More.jsx

```diff
@@ -1,2 +1,2 @@
 module.exports = require('./Highcharts.jsx');
-require("exports?Highcharts!highcharts-more");
\ No newline at end of file
+require("exports?Highcharts!higgihcharts-more");
```

Comment:
Apart from that looks good to me! 

![giphy](https://cloud.githubusercontent.com/assets/5455767/9734389/900b804a-562a-11e5-91ac-11814e5a141f.gif)